docs/cellxgene_census_docsite_data_release_info.md

```diff
@@ -1,23 +1,110 @@
 # Census data releases 
 
-**Last edited**: April, 2023.
+**Last edited**: July, 2023.
+
+**Contents**
+
+1. [What is a Census data release?](#What-is-a-Census-data-release)
+2. [List of LTS Census data releases](#List-of-LTS-Census-data-releases)
 
 ## What is a Census data release?
 
 It is a Census build that is publicly hosted online. A Census build is 
 a [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA) collection with the Census data from [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) as specified in the [Census schema](cellxgene_census_docsite_schema.md). 
 
-### Long-term supported (LTS) Cell Census releases
+### Long-term supported (LTS) Census releases
 
 To enable data stability and scientific reproducibility, [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) plans to perform regular LTS Census data releases:
 
 * Published online every six months for public access, starting on May 15, 2023.
 * Available for public access for at least 5 years upon publication.
  
+The latest LTS Census data release is the default opened by the APIs and recognized as the `census_version = "stable"`.
+
+Python
+
+```python
+import cellxgene_census
+census = cellxgene_census.open_soma()
+```
+
+R
+
+```r
+library("cellxgene.census")
+census <- open_soma()
+```
 
 ### Weekly Census release (latest)
 
 [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) ingests a handful of new datasets every week. To quickly enable access to these new data via the Census, [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) plans to perform weekly Census data releases:
 
-* Published online every Monday for public access, except for CZI holidays. 
-* Available for public access for 1 week or until the next latest release is performed, whichever is the longest.
\ No newline at end of file
+* Available for public access for 1 week or until the next latest release is performed, whichever is the longest.
+
+The weekly release can be opened by the APIs by specifying `census_version = "latest"`.
+
+Python
+
+```python
+import cellxgene_census
+census = cellxgene_census.open_soma(census_version = "latest")
+```
+
+R
+
+```r
+library("cellxgene.census")
+census <- open_soma(census_version = "latest")
+```
+
+## List of LTS Census data releases
+
+### LTS 2023-05-15
+
+Open this data release by specifying `census_version = "2023-06-28"` in future calls to `open_soma()`.
+
+#### 🔴 Erratum 🔴  
+
+There are 243,569 number of cells labelled as `is_primary_data = True` for which a fraction of them the label is incorrect. 
+
+Such label indicates that a cell is the primary representation of an observation, otherwise a cell is deemed to be a duplicate representation. Based on their count vectors, these 243,569 number of cells are represented at least twice with `is_primary_data = True`.
```

Comment:
~Such label indicates that a cell is the primary representation of an observation~ - I think this will introduce confusion (What does "primary representation" mean?).
~Based on their count vectors~ - I don't think it matters how we found them

Suggest something like...
_In order to prevent duplicate data in analyses, each observation should be marked `is_primary data = True` exactly once in the Census. Since this LTS release, 243,569 observations have been identified that are represented at least twice with `is_primary_data = True`._